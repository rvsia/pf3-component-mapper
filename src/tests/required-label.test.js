import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import RequiredLabel from '../form-fields/required-label';

describe('<RequiredLabel />', () => {
  let initialProps;
  beforeEach(() => {
    initialProps = {
      label: 'Foo',
    };
  });

  it('should render correctly', () => {
    const wrapper = mount(<RequiredLabel { ...initialProps } />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
