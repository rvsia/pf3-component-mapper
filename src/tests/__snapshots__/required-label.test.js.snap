// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RequiredLabel /> should render correctly 1`] = `
<RequiredLabel
  label="Foo"
>
  <span
    style={
      Object {
        "color": "#ff0000",
      }
    }
  >
    * 
  </span>
   
  Foo
</RequiredLabel>
`;
